### DISCUSSION TITLE
How to check if the websocket connection was successful!?

### DISCUSSION THREAD

**Discussion by Decooh** [Novage/p2p-media-loader#393] [General]
Hi guys!!
Does anyone know if this is currently possible? 🤔

**Comment by mrlika** [comment]
There is no such functionality currently. We will investigate if it is possible to add events\statuses for that.

**Reply by Decooh** [reply]
Hey.. hi @mrlika 
It's been a while.. :D

Were you able to implement it?

**Reply by mrlika** [reply]
Hi! `HlsJsP2PEngine` instance has `onTrackerError` and `onTrackerWarning` events:

```js
engine.addEventListener("onTrackerError", ({ streamType, error }) => {
  console.log(streamType, error);
});
```
